src/App.js

```diff
@@ -1,16 +1,26 @@
 import React from "react";
 import logo from "./logo.png";
 import "./App.css";
+import Clock from "./Clock.js";
```

Comment:
if you have installed eslint, you should get a warning about importing a component that you didn't use: 

<img width="485" alt="image" src="https://user-images.githubusercontent.com/40119924/190841157-34d79a62-13af-4bb3-86f4-5f00d00fd15b.png">

generally a good practice to delete dead code like this before pushing